lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -673,6 +700,29 @@ queued_next_sge(void *ref, void **address, uint32_t *length)
 	return 0;
 }
 
+static void
+queued_done_adminq(void *ref, const struct spdk_nvme_cpl *cpl)
```

Comment:
This function looks like it mostly does the same as queued_done, so it would be nice to just decrement outstanding_admin_cmds and then call queued_done to reuse the common code.